### DISCUSSION TITLE
Allow multiple users to request same media

### DISCUSSION THREAD

**Discussion by hamoudydev** [sct/overseerr#2687] [Feature Requests]
If a movie or series has already been requested by another user, other users should also get an option to request it so that they may be notified once available. After the first request it should say something along the lines of "request notification once available".

**Comment by Thunder7ga** [comment]
At least allow them to set to also be notified when it comes available.   And, allow the admin/owner to set specific media to notify specific users as well.

**Comment by Hostground** [comment]
This is definitely a missing feature.

**Comment by witterally** [comment]
Almost three years later and this is still a need! I'd love to see it added, really just so multiple users can get notifications on media availability.

**Comment by marchershey** [comment]
...and if User A requests Season 1 & 2, allow User B to request Season 3 & 4. 

Idk why I'm even commenting on this. It's not like Overseerr is active anymore...

**Comment by marclh1992** [comment]
This would be a great feature that Petio had that was really useful for popular releases that multiple people will want notifications for.
